index.html

```diff
@@ -28,17 +28,205 @@ <h1 class="greeting">I'm Ivan<br>Glad to meet <br>you!</h1>
+
+        <div class="tags">
+          <ul class="tags-ul">
+            <li class="html-tag">html</li>
+            <li class="css-tag">css</li>
+            <li class="javaScript-tag">javaScript</li>
+          </ul>
+        </div>
+
+        <div class="project">
+          <button class="project-btn">View Project</button>
+        </div>
+
+      </div>
+
+      </div>
+    </div>
+
+    <div class="tonic-1">
+
+      <div class="tonic2-img"></div>
+    <div class="text-box">
+      <div class="tonic-text">
+        <h1 class="tonic-title">Tonic</h1>
+
+        <ul class="info">
+          <li id="client">CANOPY</li>
+          <li id="role">Full stack Dev</li>
+          <li id="date">2022</li>
+        </ul>
+
+        <p>A daily selection of privately <br>personalized reads, no accounts or <br>sign-ups required.</p>
+
+        <div class="tags">
+          <ul class="tags-ul">
+            <li class="html-tag">html</li>
+            <li class="css-tag">css</li>
+            <li class="javaScript-tag">javaScript</li>
+          </ul>
+        </div>
+
+        <div class="project">
+          <button class="project-btn">View Project</button>
+        </div>
+
+      </div>
+
+      </div>
+    </div>
+
+    <div class="tonic-3">
+
+      <div class="multi2-img"></div>
+    <div class="text-box">
+      <div class="tonic-text">
+        <h1 class="tonic-title">Multi-Post <br>Stories</h1>
+
+        <ul class="info">
+          <li id="client">CANOPY</li>
+          <li id="role">Full stack Dev</li>
+          <li id="date">2022</li>
+        </ul>
+
+        <p>A daily selection of privately <br>personalized reads, no accounts or <br>sign-ups required.</p>
+
+        <div class="tags">
+          <ul class="tags-ul">
+            <li class="html-tag">html</li>
+            <li class="css-tag">css</li>
+            <li class="javaScript-tag">javaScript</li>
+          </ul>
+        </div>
+
+        <div class="project">
+          <button class="project-btn">View Project</button>
+        </div>
+
+      </div>
+
+      </div>
+    </div>
+
+  </section>
+
+  <section class="about-myself">
+
+    <div class="about-me">
+      <h1 class="about-title">About <br>Myself</h1>
+      <p class="about-descript">Hello, I'm a software developer! I can help you build a product, feature or website look at some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+      
+      <div class="social-media2">
+        <a class="socials" href="">LET'S CONNECT</a><br>
+
+        <ul class="social-links">
+          <li class="linkedin2"><a href="https://www.linkedin.com/in/ivan-ludick-11a055243/"><img src="images\Linkedin icon.jpg"></a></li>
+          <li class="gmail2"><a href="https://smiley20089@gmail.com"><img src="images\Gmail.jpg"></a></li>
+          <li class="github2"><a href="https://github.com/IvanLudick"><img src="images\GitHub.jpg"></a></li>
+        </ul>
```

Comment:
- [ Optional ] Kindly add the remaining social icons just like the figma design.

Your Social Icons look likes this : 
  
![Capture](https://user-images.githubusercontent.com/93598189/205272497-a2e2bed3-1052-411c-9938-f396c14e4f6d.PNG)

The Figma design Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/205272727-b4c0f271-0f65-4a4e-89ca-8ec01c65c20e.PNG)